spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StdCmdConfig.scala

```diff
@@ -245,6 +245,26 @@ object StdCmdConfig {
           } else {
             failure("The --is-xcom option is supported only for COBOL data format")
           })
+
+      opt[String]("cobol-encoding").optional().action((value, config) => {
+        config.copy(cobolOptions = cobolSetEncoding(config.cobolOptions, value))
+      }).text("Specify encoding of mainframe files (ascii or ebcdic)")
+        .validate(value =>
+          if (rawFormat.isDefined && rawFormat.get.equalsIgnoreCase("cobol")) {
+            success
+          } else {
+            failure("The --cobol-encoding option is supported only for COBOL data format")
+          })
+
+      opt[String]("cobol-trimming-policy").optional().action((value, config) => {
+        config.copy(cobolOptions = cobolSetTrimmingPolicy(config.cobolOptions, value))
+      }).text("Specify string trimming policy for mainframe files (none, left, right, both)")
+        .validate(value =>
+          if (rawFormat.isDefined && rawFormat.get.equalsIgnoreCase("cobol")) {
```

Comment:
won't you check if the value is one of `Seq("none", "left", "right", "both")`?

Reply:
Cobrix will validate this anyway. And if the set of trimming options is changed (new option is added for instance) in Cobrix we won't need to make changes in Enceladus.

Having said that, I can add a test for a bogus trimming policy.

Reply:
I am sorry for treating Cobrix as a random 3rd party library at the time of review 🙇‍♂️